### PR TITLE
fix: can not collect fee when position is closed

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -165,4 +165,4 @@
   "resolutions": {
     "react-error-overlay": "6.0.9"
   }
-}
\ No newline at end of file
+}
```
Comment (namgold) [inline package.json]: người thì thêm người thì xoá :pepe_angry:

Comment (namgold) [reply package.json]: should unified dis

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "@datadog/browser-rum": "^4.23.3",
     "@kyberswap/ks-sdk-classic": "^1.0.1",
     "@kyberswap/ks-sdk-core": "^1.0.1",
-    "@kyberswap/ks-sdk-elastic": "^1.0.1",
+    "@kyberswap/ks-sdk-elastic": "^1.0.2-rc",
```
Comment (namgold) [inline package.json:45]: should it be `1.1.0-rc`?

File: package.json (hunk 1 above)
Comment (viet-nv) [reply package.json]: lint auto

**Review by namgold — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (viet-nv) [reply package.json:45]: no, this is not important :pepe-sad:

**Pull request merged**